src/AddTodoForm.jsx

```diff
@@ -1,24 +1,18 @@
 import { useState } from 'react';
 
 const AddTodoForm = ({onAddTodo}) => {
-    let [todoTitle, setTodoTitle] = useState("");
+    const [todoTitle, setTodoTitle] = useState("");
 
     const handleTitleChange = (event) => {
-        let todoInput = document.querySelector("input");
-        let newTodoTitle = todoInput.value;
-        //let newTodoTitle = event.target.value;
+        const todoInput = document.querySelector("input");
```

Comment:
I'm looking between here any your other branch `lesson_1_5_b` and see that your are still referencing and updating the DOM directly. `document.querySelector` should only be used in a React application where HTML elements that need manipulated reside outside the React application or while interacting with very specific libraries that do their own DOM updates (some chart visualization libraries, interactive maps, etc.).

Remember that the value of the input is tied to the `value={todoTitle}` input attribute. That `todoTitle` is the only place where we should be reading its value and `setTodoTitle` is the only way we shoud update that value.

Here is an example what your updated code may look like: 

```jsx
const [todoTitle, setTodoTitle] = useState("");
const handleTitleChange = (event) => {
        setTodoTitle(event.target.value);
    };
//
// just focusing on the input element here & leaving out all the other jsx
//
<input type="text" name="title" id='todoTitle' value={todoTitle} onChange={handleTitleChange} />
```

Take a look closer look at the element's attributes and we can see how the the input's value is being managed by `useState`. In `<input/>`, we have 2 attributes that interest us: `value` and `onChange`. `value` is the value of the input that is currently displayed in the field. `onChange`is an event listener calls `handleTitleChange` every time its triggered. Since we are only passing in a function reference, `onChange` is going to call it and pass in an `event` object that we can work with. I would encourage you to add a console statement in `handleTitleChange`to output that event object so you can see all of the fun stuff it contains.

```javascript
//using dir instead of log is a nicer way to look at objects and arrays
console.dir(event)
```

Please reach out via Slack or on here if you need help making this update.